src/main/webapp/app/shared/badge/CountBadge.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './count-badge.scss';
+import classNames from 'classnames';
+
+export interface CountBadgeProps {
+  base: string | JSX.Element;
+  count: number;
+  hideWhenOne?: boolean;
+  max?: number;
+}
+
+const CountBadge = (props: CountBadgeProps) => {
+  const count = props.count > (props.max || Number.MAX_SAFE_INTEGER) ? `${props.max}+` : props.count;
+  const hideWhenOne = props.hideWhenOne !== undefined ? props.hideWhenOne : false;
+
+  return (
+    <div className="mr-1">
+      <div className={classNames('pt-2', hideWhenOne ? 'pr-0' : 'pr-2', 'count-badge-wrapper')}>
```

Comment:
pr-0 looks a bit off when other icons within the summary do have pr-2. Could it be only at pr-0 if no other badge uses pr-2?